ipfs-cluster-service/main.go

```diff
@@ -356,15 +371,14 @@ func initConfig(force bool, generateSecret bool, envSecret string) {
 	cfg, err := ipfscluster.NewDefaultConfig()
 	checkErr("creating default configuration", err)
 
-	if !generateSecret {
-		if len(envSecret) != 0 {
-			// read cluster secret from env variable
-			fmt.Println("Reading cluster secret from CLUSTER_SECRET environment variable.")
-			cfg.ClusterSecret, err = ipfscluster.DecodeClusterSecret(envSecret)
-		} else {
-			// get cluster secret from user
-			cfg.ClusterSecret, err = ipfscluster.DecodeClusterSecret(promptUser("Enter cluster secret (to automatically generate, rerun with --gen-secret): "))
-		}
+	if len(envSecret) != 0 {
+		// read cluster secret from env variable
+		fmt.Println("Reading cluster secret from CLUSTER_SECRET environment variable.")
+		cfg.ClusterSecret, err = ipfscluster.DecodeClusterSecret(envSecret)
+	} else if !generateSecret {
+		// get cluster secret from user
+		cfg.ClusterSecret, err = ipfscluster.DecodeClusterSecret(promptUser("Enter cluster secret (to automatically generate, rerun with --gen-secret): "))
```

Comment:
Yeah good point. I'm not sure. `--option` feels more natural to me (for long options) and `-option` feels more Go-ish. I might have been using both forms depending on the feeling of the day. I have used `--option` for these changes. Happy to settle on that and keep an eye on next doc reviews.